tarpc/examples/server_calling_server.rs

```diff
@@ -59,7 +58,7 @@ impl DoubleService for DoubleServer {
 
     fn double(self, _: context::Context, x: i32) -> Self::DoubleFut {
         async fn double(mut client: add::Client, x: i32) -> Result<i32, String> {
-            let result = await!(client.add(context::current(), x, x));
+            let result = client.add(context::current(), x, x).await;
```

Comment:
No reason for a temp variable anymore; mind inlining this line with the next? It's a good example of await chaining.

Reply:
Good catch!